lib/buildFactory.js

```diff
@@ -93,14 +95,16 @@ class BuildFactory extends BaseFactory {
             status: 'QUEUED'
         }, config);
 
+        delete modelConfig.apiUri;
+
         return getCommitSha(this.datastore, modelConfig)
             .then(sha => {
                 modelConfig.sha = sha;
 
                 return super.create(modelConfig);
```

Comment:
modelConfig includes the `username` and `executor`, I don't they they should be passed to `create`.